theano/compile/mode.py

```diff
@@ -226,6 +226,15 @@ def apply(self, fgraph):
 optdb.register('merge3', gof.MergeOptimizer(),
                100, 'fast_run', 'merge')
 
+if theano.config.check_stack_trace in ['raise', 'warn', 'log']:
+    tags = ('fast_run', 'fast_compile')
```

Comment:
Maybe name that temporary variable something that looks more private like `_tags`, or just `del tags` after calling `optdb.register`.
That code would make it usable as `theano.compile.mode.tags`, but that would not really make sense for anything else than `CheckStackTrace`.